### PR TITLE
Added examples in unsigned integers for MaxCLL, MaxFall

### PR DESCRIPTION
**Pull request by digitaltvguy**
Added examples in unsigned integers for MaxCLL, MaxFall

Also moved MaxCLL description above MaxFall to match examples and description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4127,7 +4127,7 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
                <tr>
                  <th>Actual value</th>
                  <th>Stored Decimal values</th>
-                 <th>Stored 2-byte Hex unsigned values</th>
+                 <th>Stored 4-byte Hex unsigned values</th>
```
Comment (svgeesus) [inline index.html:4130]: Shouldn't this simply say "4 bytes" and then, in the prose, 

> Each value is encoded as a [PNG four-byte unsigned integer](https://w3c.github.io/PNG-spec/#dfn-png-four-byte-unsigned-integer)

**Review by svgeesus — CHANGES_REQUESTED** [review]
4-byte hex is inconsistent with other chunk descriptions

File: index.html [hunk 2]
```diff
@@ -3685,12 +3685,13 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
...
             A <code>Matrix Coefficients</code> value other than <code>0</code> signals a transformation between colour difference
-            and RGB representations, which is not used in this International Standard since PNG images are RGB images.
+            and RGB representations, which is not used in this International Standard since PNG images are RGB images. These are decimal
+            values.  Examples below show the 1-byte unsigned equivalents.
           </aside>
 
           <p>The <code>Video Full Range Flag</code> value MUST be either <code>0</code> or <code>1</code>. A value of 1 specifies
-            that the content has a full range scaling. This is the usual value for RGB images. A value of 0 specifies that the 
-            content has a limited-range scaling factor</p>
+            that the content has a full range scaling. This is the default value for RGB images. A value of 0 specifies that the
```
Comment (svgeesus) [inline index.html:3693]:
```suggestion
            that the content has a full range scaling. This is the usual value for RGB images. A value of 0 specifies that the 
```

File: index.html [hunk 3]
```diff
@@ -4037,15 +4059,17 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
...
           <p>MaxFALL (Maximum Frame Average Light Level) uses a static metadata value to indicate the 
             maximum value of the <a>frame</a> average light level (in cd/m<sup>2</sup>, also known as nits) 
             of the entire playback sequence. MaxFALL is calculated by first averaging the decoded luminance 
             values of all the pixels in each frame, and then using the value for the frame with the highest value.</p>
              
-          <p>MaxCLL (Maximum Content Light Level) uses a static metadata value to indicate the maximum light 
-            level of any single pixel (in cd/m<sup>2</sup>, also known as nits) of the entire playback sequence. 
-            There is often an algorithmic filter to eliminate false values occurring from processing or noise 
-            that could adversely affect intended downstream tone mapping.</p>
+            The MaxCLL and MaxFALL values are encoded as <a>PNG four-byte unsigned integers</a>s,
```
Comment (chrisn) [inline index.html:4072]: Add paragraph tags.

File: index.html [hunk 4]
```diff
@@ -3699,12 +3700,12 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
...
             image</a>. Narrow range images are found in video workflows where the interpretation of sample values below reference
             black (0% signal level) or above nominal peak (100% signal level). For example, [[ITU-R-BT.709]] specifies that, for
-            10-bit coding, reference black (called black level) corresponds to code value 64 and nominal peak to code value 940.
-            In narrow range, momentary excursions defined as overshoots and undershoots exist below reference black and above nominal
-            peak in order to preserve processing artifacts caused by filtering/compression or by uncontrolled lighting without
-            clipping.  This can improve image quality during additional stages of processing and compression. The use of
-            undershoot/overshoot has also been used to preserve additional color volume (both light and color) as described in
-            [[EBU-R-103]]. [[SMPTE-RP-2077]] describes full range in more detail and includes the mapping
+            10-bit coding, reference black (called black level) corresponds to a 10-bit code value of 64 and nominal peak to a
```
Comment (chrisn) [inline index.html:3703]: There's no need to add "10-bit" here, as that's already established by the previous "for 10-bit coding"

**Review by chrisn — CHANGES_REQUESTED** [review]

**Comment by ProgramMax** [comment]
Friendly ping: There are changes requested in order for this patch to land.

**Comment by digitaltvguy** [comment]
> In our last meeting, we mentioned that the two open pull requests (this is one of them) are for Fourth Edition. But I think we said that wrong. I think this pull request should go into Third Edition, right? MaxCLL and MaxFALL are already in Third Edition.

I think it was ndwt (nominal diffuse white metadata) that we said should be in fourth edition.

File: index.html (hunk 4 above)
Comment (digitaltvguy) [reply index.html:3703]: This looks fine.

**Comment by digitaltvguy** [comment]
> In our last meeting, we mentioned that the two open pull requests (this is one of them) are for Fourth Edition. But I think we said that wrong. I think this pull request should go into Third Edition, right? MaxCLL and MaxFALL are already in Third Edition.

The "10-bit" wording was fixed.  What else needs changing?

File: index.html (hunk 1 above)
Comment (ProgramMax) [reply index.html:4125]: I'm updating this PR right now to land it.
I was about to comment "Should this mention network byte order? That gets wordy. Maybe link to the PNG four-byte thing?" Then I saw your comment.

But I think you may have meant something different by 'prose'. You don't mean the table header text here should link, right? Rather, before the tables there should be some wording to explain that it is the PNG four-byte and link there?

Comment (ProgramMax) [reply index.html]: I fixed the rest and looped back to this. I was going to add your suggested text just before the examples. But I notice right there it already explains which values are PNG two-byte unsigned integers and which are PNG four-byte unsigned integers.

I think we can leave that existing text.

**Comment by ProgramMax** [comment]
I believe this is now ready to land.
I'll add my approval. But please review my changes and confirm you're happy with them.

**Review by ProgramMax — APPROVED** [review]

File: index.html (hunk 4 above)
Comment (digitaltvguy) [reply index.html]: I think I've merged this

**Review by svgeesus — APPROVED** [review]
Approved. @chrisn does this satisfy your requested changes?

**Comment by ProgramMax** [comment]
Friendly ping @chrisn -- Does this work for you?

**Comment by chrisn** [comment]
Sorry for the delay. Yes, all looks good.

**Review by chrisn — APPROVED** [review]

**Pull request merged**